src/zhinst/toolkit/nodetree/node.py

```diff
@@ -266,15 +266,21 @@ class Node:
     >>> nodetree.demods[0].freq(2000)
     >>> nodetree.demods[0].freq()
     2000
+
     >>> nodetree.demods["*"].freq(3000)
     >>> nodetree.demods["*"].freq()
     {
-        /dev1234/demods/0/freq: 3000
-        /dev3036/demods/1/freq: 3000
-        /dev3036/demods/2/freq: 3000
-        /dev3036/demods/3/freq: 3000
+        '/dev1234/demods/0/freq': 3000
+        '/dev3036/demods/1/freq': 3000
+        '/dev3036/demods/2/freq': 3000
+        '/dev3036/demods/3/freq': 3000
```

Comment:
```suggestion
        '/dev1234/demods/1/freq': 3000
        '/dev1234/demods/2/freq': 3000
        '/dev1234/demods/3/freq': 3000
```

pls fix my mistakes 😇